ironfish-rust-nodejs/Cargo.toml

```diff
@@ -30,6 +30,7 @@ base64 = "0.13.0"
 fish_hash = "0.1.0"
 ironfish = { path = "../ironfish-rust" }
 ironfish_mpc = { path = "../ironfish-mpc" }
+ironfish-frost = { git = "https://github.com/iron-fish/ironfish-frost.git" }
```

Comment:
do we need this dependency?

Reply:
I use `use ironfish::frost::Identifier;`

Reply:
I tried removing it and it works without it. I'll push the change. I could have sworn that i wasn't able to import it